migaloo/assetlist.json

```diff
@@ -83,6 +83,26 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/fable.svg"
       }
-    }
+    },
+      {
+        "description": "boneLUNA are autocompounding LUNA tokens",
+        "denom_units": [
+          {
+            "denom": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+            "exponent": 0
+          },
+          {
+            "denom": "bluna",
+            "exponent": 6
+          }
+        ],
+        "base": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+        "name": "boneLuna",
+        "display": "bluna",
+        "symbol": "bLUNA",
+        "logo_URIs": {
```

Comment:
could you please add the trace from terra2?

Reply:
You can see how Carbon registered LUNA here: https://github.com/cosmos/chain-registry/blob/c9f0a68c0ea92469e56c5a6bb7f2fba029aa9fc6/carbon/assetlist.json#L344-L381
You'd just instead use the connection numbers from the migaloo-terra2 ibc file.